.settings/org.eclipse.wst.jsdt.ui.superType.container

```diff
@@ -0,0 +1 @@
+org.eclipse.wst.jsdt.launching.JRE_CONTAINER
```

Comment:
Remove and ignore this as well?

Reply:
I actually like having these checked in because it makes it easier for another developer to pull into Eclipse.

Reply:
Ok